recording_server/src/streaming.ts

```
@@ -26,18 +26,22 @@ export class Streaming {
     private outputUrl: string | undefined
     private botId: string
 
-    // État du streaming
+    // Streaming state management
     private isInitialized: boolean = false
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)